src/index.js

```diff
@@ -0,0 +1,12 @@
+import toDos from './modules/items.js';
+import './styles/style.css';
+
+const toDosDiv = document.getElementById('todo-list');
+const toDosLists = (myToDos) => <div class = "toDo-list">
+  <ul>
+   <input type="checkbox"></input>
+    <span>${myToDos.description}</span>
+  </ul>
+</div>;
+toDosDiv.innerHTML = `
+${toDos.map(toDosLists).join('')}`;
```

Comment:
- [ ] Kindly populate the UI with the array of data you created. You will need to loop the array and you can try to follow this format after looping so as to render it.

```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```